configs/sites/noaa-aws/packages.yaml

```diff
@@ -0,0 +1,187 @@
+packages:
+  all:
+    compiler:: [intel@2021.5.0]
+    providers:
+      mpi:: [intel-oneapi-mpi@2021.5.1]
+  mpi:
+    buildable: False
+  intel-oneapi-mpi:
+   externals:
+   - spec: intel-oneapi-mpi@2021.5.1%intel@2021.5.0
+     prefix: /contrib/spack-stack/apps/intel-oneapi
+     modules:
+     - impi/2022.1.2
+  autoconf:
+    externals:
+    - spec: autoconf@2.69
+      prefix: /usr
+  automake:
+    externals:
+    - spec: automake@1.13.4
+      prefix: /usr
+  bash:
+    externals:
+    - spec: bash@4.2.46
+      prefix: /usr
+  berkeley-db:
+    externals:
+    - spec: berkeley-db@5.3.21
+      prefix: /usr
+  bison:
+    externals:
+    - spec: bison@3.0.4
+      prefix: /usr
+  bzip2:
+    externals:
+    - spec: bzip2@1.0.6
+      prefix: /usr
+  cmake:
+    externals:
+    - spec: cmake@2.8.12.2
+      prefix: /usr
+  cpio:
+    externals:
+    - spec: cpio@2.11
+      prefix: /usr
+  diffutils:
+    externals:
+    - spec: diffutils@3.3
+      prefix: /usr
+  doxygen:
+    externals:
+    - spec: doxygen@1.8.5~graphviz~mscgen
+      prefix: /usr
+  file:
+    externals:
+    - spec: file@5.11
+      prefix: /usr
+  findutils:
+    externals:
+    - spec: findutils@4.5.11
+      prefix: /usr
+  flex:
+    externals:
+    - spec: flex@2.5.37+lex
+      prefix: /usr
+  gawk:
+    externals:
+    - spec: gawk@4.0.2
+      prefix: /usr
+  gettext:
+    externals:
+    - spec: gettext@0.19.8.1
+      prefix: /usr
+  ghostscript:
+    externals:
+    - spec: ghostscript@9.25
+      prefix: /usr
+  git:
+    externals:
+    - spec: git@2.23.3~tcltk
+      prefix: /usr
+  git-lfs:
+    externals:
+    - spec: git-lfs@3.1.2
+      prefix: /contrib/spack-stack/apps/utils
+  gmake:
+    externals:
+    - spec: gmake@3.82
+      prefix: /usr
+  groff:
+    externals:
+    - spec: groff@1.22.2
+      prefix: /usr
+  hwloc:
+    externals:
+    - spec: hwloc@1.11.8
+      prefix: /usr
+  openjdk:
+    externals:
+    - spec: openjdk@1.8.0_312-b07
+      prefix: /usr
+  krb5:
+    externals:
+    - spec: krb5@1.15.1
+      prefix: /usr
+  libfuse:
+    externals:
+    - spec: libfuse@2.9.2
+      prefix: /usr
+  libtool:
+    externals:
+    - spec: libtool@2.4.2
+      prefix: /usr
+  lustre:
+    externals:
+    - spec: lustre@2.10.8
+      prefix: /usr
+  m4:
+    externals:
+    - spec: m4@1.4.16
+      prefix: /usr
+  ncurses:
+    externals:
+    - spec: ncurses@6.3.20211021+termlib abi=6
+      prefix: /contrib/spack-stack/apps/miniconda/3.9.7
+    - spec: ncurses@6.0.20170212+termlib abi=6
+      prefix: /usr
+  openssh:
+    externals:
+    - spec: openssh@7.4p1
+      prefix: /usr
+  openssl:
+    externals:
+    - spec: openssl@1.1.1n
```

Comment:
I would keep the miniconda entry and remove the other one, to avoid that you get multiple versions of ncurses used in the stack.

Reply:
For OpenSSL I think it's better to use the system version. In case of any security vulnerability it's going to be updated by the OS, and maybe not my Python install.